tests/vmi_configuration_test.go

```diff
@@ -816,7 +816,7 @@ var _ = Describe("Configurations", func() {
 				}
 				cpuVmi.Spec.Domain.Resources = v1.ResourceRequirements{
 					Requests: kubev1.ResourceList{
-						kubev1.ResourceMemory: resource.MustParse("64M"),
+						kubev1.ResourceMemory: resource.MustParse("128M"),
```

Comment:
hm From where coming memory change?

Reply:
I changed it back, since that is not a proper fix. It looks like we don't request enough overhead. Our dedicated CPU VMIs have a QoS class of guaranteed, which also means that they are not allowed to use more memory than the requested memory + overhead. I have seen that they get sometimes killed because of the OOM killer. Might be that our overhead calculation is not good enough. @MarSik FYI. Do we already have some libvirt/qemu numbers in order to see if we should add more overhead?